quantum/debounce/sym_defer_g.c

```diff
@@ -24,6 +24,12 @@ When no state changes have occured for DEBOUNCE milliseconds, we push the state.
 #    define DEBOUNCE 5
 #endif
 
+// Maximum debounce: 255ms
+#if DEBOUNCE > UINT8_MAX
+#    undef DEBOUNCE
+#    define DEBOUNCE UINT8_MAX
```

Comment:
Not sure it's the best option, but maybe a pragma message should be here to notify the user, if not an error outright.

Reply:
Sure. I'm just following what the other implementations do here.

I can make that a general improvement in the PR that will follow.

Reply:
Ah, yeah, the other files are doing the same.  In that case, it should be okay here, as I'd rather it be consistent.